src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -31,8 +31,11 @@ def load_arguments(self, _):
             '--name', '-n'], help='Name of Azure Spring Cloud.')
 
     with self.argument_context('spring-cloud create') as c:
-        c.argument('location', arg_type=get_location_type(
-            self.cli_ctx), validator=validate_location)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=validate_location)
+        c.argument('sku', type=str, validator=validate_sku, help='Name of SKU, the value is "Basic" or "Standard"')
```

Comment:
Why not use `get_enum_type` for sku?

Reply:
is it able to customize the CLI error in get_enum_type?

raise CLIError("The pricing tier only accept value [Basic, Standard]")

Reply:
The error message of get_enum_type cannot be customized and will tell users to see --help for the allowed values.